### ISSUE TITLE
cannot open ImageNet dataset with .sh file

### ISSUE THREAD

**Issue by germanjke** [pytorch/examples#700]
Hello i can't open dataset in this code
I've open it in GitBash (my os is windows10)
Script is launching, and folders creating but later
_mv ILSVRC2012_val_00000001.JPEG n01751748_ when commands like this launch - my console give me: `mv: cannot stat no such file or directory`

**Comment by nickxtsui** [comment]
Same here. Please help. Thanks.

**Comment by limkeunhak** [comment]
The image files do not exist in that path. (https://raw.githubusercontent.com/soumith/imagenetloader.torch/master/valprep.sh)

If you want to run the APP, you need to download the image file from elsewhere. 
You can get it from the path below.

- http://www.image-net.org/

@soumith
Your script doesn't work. Could you help or any comments?

**Comment by soumith** [comment]
I've clarified language in README that the imagenet dataset has to be downloaded, and then the `valprep.sh` only reorganizes the validation set into labeled folders.

**Comment by soumith** [comment]
clarified via the commit https://github.com/pytorch/examples/commit/e9e76722dad4f4569651a8d67ca1d10607db58f9

**Comment by nk0307** [comment]
same question
